apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss

```diff
@@ -104,29 +104,29 @@ i.close.icon.clear:hover::after {
 }
 
 .selected-val {
-    border: 1px solid $light-gray !important;
+    border: 1px solid $gainsboro !important;
     padding-right: 5px !important;
     background: $white !important;
 }
 
 .header.open > input[type="text"] {
-    border-color: $light-gray;
+    border-color: $gainsboro;
 }
 
 .header .bdr > input[type="text"] {
     border: 0;
 }
 .header .bdr > input[type="text"]::-webkit-input-placeholder {
-    color: $nobel !important;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine